docs/architecture/tls13-support.md

```diff
@@ -182,36 +139,21 @@ MVP definition
   | MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED     | n/a     |
   |                                          |         |
   | MBEDTLS_PSA_CRYPTO_C                     | no (1)  |
-  | MBEDTLS_USE_PSA_CRYPTO                   | no      |
+  | MBEDTLS_USE_PSA_CRYPTO                   | yes     |
 
   (1) These options must remain in their default state of enabled.
   (2) Key exchange configuration options for TLS 1.3 will likely to be
       organized around the notion of key exchange mode along the line
       of the MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_NONE/PSK/PSK_EPHEMERAL/EPHEMERAL
       runtime configuration macros.
```

Comment:
If the note for `MBEDTLS_KEY_EXCHANGE_PSK_ENABLED` has changed from `(2)` to `(1)`, I don't think there's any reference to note `(2)` left, so

```suggestion
```

Reply:
The change from (2) to (1) for MBEDTLS_KEY_EXCHANGE_PSK_ENABLED was an error. Changed back.